components/Bubbles/BubblesGame.js

```diff
@@ -107,6 +126,15 @@ class BubblesGame extends React.Component {
     });
     this.setState({bubbleArray: remainingBubbles});
   }
+
+  onCharacterTweenFinish (characterUID) {
```

Comment:
if only monster character, or only some single character, we can get rid of switch statement and passed in arg.